docker-compose.monitoring.yml

```diff
@@ -19,5 +19,5 @@ services:
     command:
       - --config.file=/etc/prometheus/prometheus.yml
     volumes:
-      - ./data/prometheus:/prometheus
+      - ./data/prometheus:/data
```

Comment:
I don't think Prometheus stores it's data files within `/data` within the container.

Reply:
https://github.com/prometheus/prometheus/blob/master/Dockerfile#L23

Reply:
Well, that looks right, but when I tried again with /prometheus, I get the following error:
```
level=error ts=2020-05-06T05:42:53.954Z caller=query_logger.go:87 component=activeQueryTracker msg="Error opening query log file" file=data/queries.active err="open data/queries.active: no such file or directory"
panic: Unable to create mmap-ed active query log
```
This is from where I determined that it actually tries to search in /data instead of /prometheus. I'll leave it up to you if you change this or not.